superdesk/commands/index_from_mongo.py

```diff
@@ -67,6 +68,9 @@ def copy_resource(cls, resource, page_size):
                 else:
                     break
 
+            if last_id:
+                print(last_id)
```

Comment:
yep, I will print last_id after the filter we are fetching from items like `last_id = items[-1][config.ID_FIELD]` so that we will get the last_id of the items processed every time.